ws-android/Dockerfile

```diff
@@ -5,65 +5,180 @@ FROM cloud9/workspace
 # https://github.com/lukin0110/docker-android-build
 MAINTAINER Jeremy Ellis <keyfreemusic@gmail.com>
 
+# -------------------------- Startup Stuff -----------------------------------------
+# move files from the files folder to the docker container
 ADD ./files/workspace /home/ubuntu/workspace
 
-RUN chmod -R g+w /home/ubuntu/workspace && \
+# Give owner and ubuntu group: read,write,execute permissions, others read only
+# Change owner to ubuntu
+RUN chmod -R 774 /home/ubuntu/workspace && \
     chown -R ubuntu:ubuntu /home/ubuntu/workspace
     
+    
+
+# Setup a bash file to auto load environment variables when in cloud 9
+# Add to this file for every environment variable and Path used
+RUN touch /etc/profile.d/android.sh && \
+    chmod -R 775 /etc/profile.d/android.sh && \
+    chown -R ubuntu:ubuntu /etc/profile.d/android.sh && \
+    printf "#!/bin/bash\n\n" > /etc/profile.d/android.sh  && \
+    printf "echo 'Setting up environment variables'\n" >>               /etc/profile.d/android.sh    && \
+    printf "JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64\n" >>           /etc/profile.d/android.sh    && \
+    printf "PATH \$PATH:\$JAVA_HOME/bin\n\n" >>                         /etc/profile.d/android.sh    
+
+
+# These commands just let us test the enviornment before submitting to cloud 9    
+ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
```

Comment:
These need to just be in the profile.d, and will be sourced when check-environment is run, ENV blocks shouldn't be needed